dev/ci/docker/bionic_coq/hooks/post_push

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
Is this a git hook or some other meaning of hook?

Reply:
This is a `hub.docker.com` hook, see the README for the link to the docs. I'll do a pass tho.